Cod3rsGrowth.Web/wwwroot/Webapp/model/formatter.js

```diff
@@ -0,0 +1,298 @@
+
+        alterarStatusDoPagamento: function (StatusPagamento) {
+            const oResourceBundle = this.getOwnerComponent().getModel("i18n").getResourceBundle();
+            switch (StatusPagamento) {
+                case true:
+                    return oResourceBundle.getText("Status.PagamentoConcluido");
+                case false:
+                    return oResourceBundle.getText("Status.PagamentoPendente");
+                default:
+                    return StatusPagamento;
+            }
+        },
+
+        formatarFlex(Flex) {
+            const oResourceBundle = this.getOwnerComponent().getModel("i18n").getResourceBundle();
+            switch (Flex) {
+                case true:
+                    return oResourceBundle.getText("Flex.Sim");
+                case false:
+                    return oResourceBundle.getText("Flex.Nao");
+                default:
+                    return Flex;
+            }
+        },
+        formatarMarcaParaInteiro(Marca){
+            switch (Marca) {
+                case "Audi":
+                    return 0;
+                case 'Bentley':
+                    return 1;
+                case "BMW":
+                    return 2;
+                case "Bugatti":
+                    return 3;
+                case "Chevrolet":
+                    return 4;
+                case "Ferrari":
+                    return 5;
+                case "Fiat":
+                    return 6;
+                case "Ford":
+                    return 7;
+                case "Honda":
+                    return 8;
+                case "Hyundai":
+                    return 9;
+                case "Jaguar":
+                    return 10;
+                case "Kia":
+                    return 11;
+                case 'Lamborghini':
+                    return 12;
+                case "Land Rover":
+                    return 13;
+                case "Maserati":
+                    return 14;
+                case "Mercedes":
+                    return 15;
+                case "Mitsubishi":
+                    return 16;
+                case "Nissan":
+                    return 17;
+                case "Peugeot":
+                    return 18;
+                case "Porsche":
+                    return 19;
+                case "Renault":
+                    return 20;
+                case "Rolls Royce":
+                    return 21;
+                case "Subaru":
+                    return 22;
+                case "Tesla":
+                    return 23;
+                case "Toyota":
+                    return 24;
+                case "Volkswagen":
+                    return 25;
+                case "Todos":
+                    return null;
+                default:
+                    return Marca;
+            }
+        },
+        formatarMarca(Marca) {
+            const oResourceBundle = this.getOwnerComponent().getModel("i18n").getResourceBundle();
```

Comment:
formatarMarca(Marca) {
    const oResourceBundle = this.getOwnerComponent().getModel("i18n").getResourceBundle();

    const marcas = {
        0: "Audi",
        1: "Bentley",
        2: "BMW",
        3: "Bugatti",
        4: "Chevrolet",
        5: "Ferrari",
        6: "Fiat",
        7: "Ford",
        8: "Honda",
        9: "Hyundai",
        10: "Jaguar",
        11: "Kia",
        12: "Lamborghini",
        13: "LandRover",
        14: "Maserati",
        15: "Mercedes",
        16: "Mitsubishi",
        17: "Nissan",
        18: "Peugeot",
        19: "Porsche",
        20: "Renault",
        21: "RollsRoyce",
        22: "Subaru",
        23: "Tesla",
        24: "Toyota",
        25: "Volkswagen"
    };

    return oResourceBundle.getText(`Marca.${marcas[Marca] || Marca}`);
}